README.md

```diff
@@ -1,3 +1,5 @@
+# This mod is deprecated due to Papermerge, Paperless-ng and Paperless-ngx having been deprecated
```

Comment:
Should this say "due to our Papermerge, Paperless-ng, and Paperless-ngx containers having been deprecated"?

Reply:
Yup